gapic-generator/lib/gapic/schema/wrappers.rb

```diff
@@ -347,6 +347,18 @@ def full_name
         @address.join "."
       end
 
+      # @private
+      # Override this to potentially add a google.api.api_version breadcrumb to
+      # client documentation.
+      def docs_leading_comments disable_xrefs: false, transport: nil
+        str = super(disable_xrefs: disable_xrefs, transport: transport)
+        if !str || !api_version || api_version.empty?
```

Comment:
`api_version.nil?` instead of checking for falsness

Reply:
Cool! Done